docs/LearningOutcomes.adoc

```diff
@@ -146,7 +146,7 @@ This learning outcome is covered by the exercise in `[LO-Refactor]`
 Note: this exercise covers two other Learning Outcomes: `[LO-CodingStandard]`, `[LO-CodingBestPractices]`
 
 * Improve the code in the following ways,
-** Fix https://github.com/oss-generic/process/blob/master/codingStandards/CodingStandard-Java.md[coding standard]
+** Fix https://github.com/oss-generic/process/blob/master/codingStandards/CodingStandard-Java.adoc[coding standard]
```

Comment:
BTW, there is a nicer-looking html version of the coding standard. We can point to that one instead :-)

Reply:
Missed this comment, swapping it over